ComfyUI/execution.py

```
@@ -344,94 +353,102 @@ def execution_block_cb(block):
-            "node_id": real_node_id,
-            "exception_message": str(ex),
-            "exception_type": exception_type,
-            "traceback": traceback.format_tb(tb),
-            "current_inputs": input_data_formatted
+            # TIMING START
+            start_time = time.perf_counter()
+            output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
+            end_time = time.perf_counter()
+            elapsed_time = end_time - start_time
+
+            CSV_LOCK = threading.Lock()  
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)